system/Debug/Toolbar.php

```diff
@@ -406,58 +406,53 @@ public function prepare(?RequestInterface $request = null, ?ResponseInterface $r
 
     /**
      * Inject debug toolbar into the response.
+     *
+     * @codeCoverageIgnore
      */
     public function respond()
     {
         if (ENVIRONMENT === 'testing') {
             return;
         }
 
-        // @codeCoverageIgnoreStart
         $request = Services::request();
 
-        // If the request contains '?debugbar then we're
-        // simply returning the loading script
```

Comment:
As someone largely unfamiliar with the Debug Bar I find the inline comments in here helpful. Did they seem too obvious to you? Or other issue?

Reply:
If you will look at the code, the inline comments tell you _what_ it is doing instead of _why_ it is doing it. So, stating the obvious is too much noise here.

Reply:
At first, I thought they were obvious. 

But after I read carefiully, I found that I didn't get the concept `the loading script` and `the entire debugbar` without the comments.

Reply:
@paulbalandan I appreciate the rubric, I will adopt that for my own comment analysis - but I would also suggest that the *what* can be helpful when the code is a bit obscure. If both kenjis and I found the comments helpful to decipher the content they might be worth keeping.